react/components/AutocompleteResults/index.js

```diff
@@ -39,8 +39,8 @@ const getLinkProps = element => {
     const paramForSearchTracking = '&_c=' + terms[0]
 
     page = 'store.search'
-    params = { term: terms[0] }
-    query = `map=c,ft&rest=${terms.slice(1).join(',')}` + paramForSearchTracking
+    params = { term: terms.join('/') }
+    query = `map=c,ft` + paramForSearchTracking
```

Comment:
Can you explain these changes? What was previously in `terms[0]`? Why now you can `.join('/')`?

Reply:
I'm reproducing this behavior.

https://github.com/vtex-apps/store/blob/ac7eb82336956672c62949e1391e24f915497df5/react/SearchContext.js#L79

The code concatenates `query` and `rest`.